src/v2/view-builder/views/SelectAuthenticatorVerifyView.js

```diff
@@ -14,12 +16,32 @@ export const Body = BaseForm.extend({
     if (this.isPasswordRecoveryFlow())  {
       return loc('oie.password.reset.verification', 'login');
     }
+    if (this.isUnlockSuccess()) {
+      const container = '.o-form-error-container';
+      const text = loc('oie.select.authenticators.verify.subtitle', 'login');
+      this.add(`<div class="ion-messages-container"><p>${text}</p></div>`, container);
+      return;
+    }
     return loc('oie.select.authenticators.verify.subtitle', 'login');
   },
   isPasswordRecoveryFlow() {
     return this.options.appState.get('isPasswordRecovery');
   },
   noButtonBar: true,
+  showMessages() {
+    if (this.isUnlockSuccess()) {
+      let options = {};
+      options.subtitle = loc('oie.selfservice.unlock_user.landing.to.app.success.message', 'login');
+      options.type = 'success';
```

Comment:
IIRC, when Idx returns the message object, it has a `class` property(possible value: info/success/warning) which can be used to style callout. but I am not sure it is usable here.

Reply:
Good callout - iirc I believe when Vicente first started working on this, that was the first thing he tried but it wasn't working as intended